### ISSUE TITLE
× symbol not shown in PGFPlots with scientific formatter

### ISSUE THREAD

**Issue by daschw** [JuliaPlots/Plots.jl#1464]
```julia
using Plots
pgfplots()
plot(rand(10), formatter = :scientific)
```
![pgfplots_scientific](https://user-images.githubusercontent.com/16589944/38053516-97940420-32d4-11e8-86a7-3c36e83d6e5e.png)
cc: @apalugniok

**Comment by apalugniok** [comment]
Mines working fine version info?

**Comment by daschw** [comment]
```julia
julia> versioninfo()
Julia Version 0.6.2
Commit d386e40 (2017-12-13 18:08 UTC)
Platform Info:
  OS: Linux (x86_64-pc-linux-gnu)
  CPU: Intel(R) Core(TM) i5-6200U CPU @ 2.30GHz
  WORD_SIZE: 64
  BLAS: libopenblas (DYNAMIC_ARCH NO_AFFINITY Haswell)
  LAPACK: libopenblas
  LIBM: libm
  LLVM: libLLVM-3.9.1 (ORCJIT, skylake)

julia> Pkg.status.(["Plots", "PGFPlots"])
 - Plots                         0.15.1+            master
 - PGFPlots                      2.2.2
```

**Comment by apalugniok** [comment]
Odd here's mine 
```
julia> Pkg.status.(["Plots", "PGFPlots"])
versioninfo()
 - Plots                         0.15.0+            master
 - PGFPlots                      2.2.2
2-element Array{Void,1}:
 nothing
 nothing

julia> versioninfo()
Julia Version 0.6.0
Commit 903644385b (2017-06-19 13:05 UTC)
Platform Info:
  OS: macOS (x86_64-apple-darwin13.4.0)
  CPU: Intel(R) Core(TM) i5-4308U CPU @ 2.80GHz
  WORD_SIZE: 64
  BLAS: libopenblas (USE64BITINT DYNAMIC_ARCH NO_AFFINITY Haswell)
  LAPACK: libopenblas64_
  LIBM: libopenlibm
  LLVM: libLLVM-3.9.1 (ORCJIT, haswell)
```
does the × show up in the output `.tex` file?

**Comment by daschw** [comment]
yes:
```
xticklabels = {$2×10^{0}$,$4×10^{0}$,$6×10^{0}$,$8×10^{0}$,$1×10^{1}$}
```

**Comment by daschw** [comment]
This happens for me on both, Linux and Windows.

**Comment by apalugniok** [comment]
Weird, I currently have no idea why this might be happening 😟

**Comment by mmikhasenko** [comment]
I am wondering, how latex is supposed to understand `×`? Should not it be `\times`?
The plotting in the scientific format works for me. The `.tex`-file also has `×` as @daschw showed.

But when I compile `tex`-document with this plot (`\input{test.tex}`) with either `pdflatex` or `lualatex`, the `×`-symbols do not appear.

**Comment by apalugniok** [comment]
I think the `\endcsname` error, that prevented us from using `\times`, was fixed by #1401. So we should be able to replace the `×` by `\times` and then this should no longer be an issue.
